src/test/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -71,11 +71,8 @@ class BasicWorkflow extends FunSpec with Matchers {
         val javaSearchSymbol = project.rpcPublicSymbolSearch(List("java", "io", "File"), 2)
         javaSearchSymbol match {
           case SymbolSearchResults(List(
-            TypeSearchResult("java.awt.image.ImageObserver", "ImageObserver", 'class,
```

Comment:
@jschlather Fair point - @fommil @aemoncannon  - Are we missing something subtle on the expected behaviour of this search method?